postcss.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  plugins: [
+    require('tailwindcss'),
+    require('autoprefixer'),
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'require' is not defined  no-undef